packages/plugins/graphql/hooks/graphql/index.js

```diff
@@ -73,6 +73,7 @@ module.exports = strapi => {
     },
 
     initialize() {
+      const nexusSchema = strapi.plugins.graphql.services.v2.generateSchema();
```

Comment:
you can clean this up right now and remove the old schema and remove the v2 and instead move the old code into a abcd.old or abcd.bak folder

Reply:
Yep, that will be clearer for sure. Done in bdf6ccc98a348f47dd2886166bac5726b7cc540e